deploy/crds/wmc.openshift.io_windowsmachineconfigs_crd.yaml

```diff
@@ -70,8 +61,8 @@ spec:
                 the OpenShift cluster
               type: integer
           required:
-          - aws
-          - azure
```

Comment:
Isn't one of aws or azure cloud provider information is required here?

Reply:
We'll have the validation [here](https://issues.redhat.com/browse/WINC-277) as it's not a static validation. I am not sure, if we can enum validations statically in yaml.